src/core/api.h

```diff
@@ -275,5 +275,6 @@ QUIC_STATUS
 QUIC_API
 MsQuicConnectionCertificateValidationComplete(
     _In_ _Pre_defensive_ HQUIC Handle,
-    _In_ BOOLEAN Result
+    _In_ BOOLEAN Result,
+    _In_ QUIC_TLS_ALERT_CODES TlsAlert
```

Comment:
Could annotate this such that it's only read when Result is false.

Reply:
I put a comment in msquic.h already. I think that's the best place.